src/spatial/models/heatmapModel.py

```
@@ -0,0 +1,799 @@
+                grid_id = grid.get("_id")
+                if not shape_geojson or not grid_id:
+                    continue
+                try:
+                    geometry = shape(shape_geojson)
+                    records.append({
+                        "id": grid_id,
+                        "name": grid.get("name"),
+                        "admin_level": grid.get("admin_level"),
+                        "geometry": geometry,
+                    })
+                except Exception as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)